pkgs/applications/graphics/unigine_heaven/default.nix

```
@@ -0,0 +1,95 @@
+{ lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  libX11,
+  libXext,
+  libXrandr,
+  freetype,
+  fontconfig,
+  libXrender,
+  libXinerama,
+  autoPatchelfHook,
+  libglvnd,
+  openal,
+}:
+
+let
+  version = "4.0";
+
+  arch = if stdenv.hostPlatform.system == "x86_64-linux" then
+    "x64"
+  else if stdenv.hostPlatform.system == "i686-linux" then
+    "x86"
+  else
+    throw "Unsupported platform ${stdenv.hostPlatform.system}";
+
+in
+
+stdenv.mkDerivation {
+  pname = "Unigine_Heaven";
```

```suggestion
  pname = "unigine-heaven";
```

To match the existing valley benchmark package.